sklearn/_loss/tests/test_loss.py

```diff
@@ -237,37 +237,48 @@ def test_loss_on_specific_values(loss, y_true, raw_prediction, loss_true):
 
 
 @pytest.mark.parametrize("loss", ALL_LOSSES)
+@pytest.mark.parametrize("readonly_memmap", [False, True])
 @pytest.mark.parametrize("dtype_in", [np.float32, np.float64])
 @pytest.mark.parametrize("dtype_out", [np.float32, np.float64])
 @pytest.mark.parametrize("sample_weight", [None, 1])
 @pytest.mark.parametrize("out1", [None, 1])
 @pytest.mark.parametrize("out2", [None, 1])
 @pytest.mark.parametrize("n_threads", [1, 2])
-def test_loss_dtype(loss, dtype_in, dtype_out, sample_weight, out1, out2, n_threads):
-    """Test acceptance of dtypes in loss functions.
+def test_loss_dtype_readonly(
+    loss, readonly_memmap, dtype_in, dtype_out, sample_weight, out1, out2, n_threads
+):
+    """Test acceptance of dtypes and readonly arrays in loss functions.
```

Comment:
As this test for both readonly and write-able arrays, this patch on the test name and top of the doc-string might be reverted.

Reply:
What exactly do you propose?

Reply:
Something like:
```suggestion
def test_loss_dtype(
    loss, readonly_memmap, dtype_in, dtype_out, sample_weight, out1, out2, n_threads
):
    """Test acceptance of dtypes and readonly and write-able arrays in loss functions.
```

This is more of a nitpick that you can ignore.